html/components/SimpleHtml.js

```diff
@@ -71,6 +71,32 @@ class SimpleHtml extends PureComponent {
     return false;
   }
 
+  /**
+   * Removes child break node in case when it's positioned at the end
+   * of parent div node and div has content in it.
+   * Example:
+   * <div>first line<br></div>
+   * <div>second line<br></div>
+   *
+   * If br is left as is, <Html> will show empty line under div, not just
+   * break into new line
+   */
+  alterChildren(node) {
+    const { children, name } = node;
+
+    if ((name === 'div' || name === 'p') && children && children.length > 1) {
```

Comment:
Only for `div` & `p`?

Reply:
Headings have top and bottom "margins" in editor & on mobile, so they're good.
Everything looks the same, on editor & mobile now.

Reply:
Not sure if you get notified on tagging you when I edit the comment, so I'll leave this here :P 

![image](https://user-images.githubusercontent.com/57353746/116541716-1d2a6d00-a8ec-11eb-967c-b3d26340b1e0.png)